logger.js

```diff
@@ -268,3 +276,4 @@ module.exports.stdSerializers = {
 module.exports.startTime = startTime
 module.exports.default = pinoLogger
 module.exports.pinoHttp = pinoLogger
+module.exports.defaultReqIdGenFactory = defaultReqIdGenFactory
```

Comment:
This should probably exposed to the tests 🤔 
Maybe using a symbol or separate file?